diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/comments/HeaderCommentRule.kt

```diff
@@ -172,10 +172,10 @@ class HeaderCommentRule(private val configRules: List<RulesConfig>) : Rule("head
                 node.addChild(PsiWhiteSpaceImpl(newLines), node.firstChildNode)
                 node.addChild(LeafPsiElement(BLOCK_COMMENT,
                     """
-                            /*
-                             * $copyrightText
-                             */
-                        """.trimIndent()),
+/*
+    ${handleMultilineCopyright(copyrightText)}
```

Comment:
As I said earlier, I think we should only prepend spaces in case there are no leading spaces in `copyrightText`. Yml gets an option to provide spaces explicitly, so if there are spaces already, the user must be expecting them to be kept.